### ISSUE TITLE
Possible issue with webpack config templates

### ISSUE THREAD

**Issue by Darkle** [NativeScript/nativescript-dev-webpack#963]
On this line: https://github.com/NativeScript/nativescript-dev-webpack/blob/master/templates/webpack.javascript.js#L20

it says `const platform = env && (env.android && "android" || env.ios && "ios");`

A non empty string will always evaluate to true, so should that instead be: 
`const platform = env && (env.android === "android" || env.ios === "ios");`?

**Comment by DimitarTachev** [comment]
Hi @Darkle,

Thanks for sharing your feedback!

In general, the platform property is always populated by the NativeScript CLI and **the end-users don't need pass it**.

The current setup is meant to support both the `"android"` or `"ios"` values from the CLI and any values for debugging purpose. For example, when I want to debug the Webpack compilation for iOS, I usually use: `node --inspect-brk ./node_modules/.bin/webpack --env.ios` (without specifiying a value for `env.ios`).

In other words, if this line does not cause any issues, we prefer to keep it in this way for faster debugging.

**Comment by Darkle** [comment]
Ok, feel free to close.
